lib/handlers/quote/config/quote-speed-config.ts

```diff
@@ -0,0 +1,35 @@
+import { MapWithLowerCaseKey, ProtocolPoolSelection } from '@uniswap/smart-order-router'
+
+export type QuoteSpeedConfig = {
+  v2PoolSelection?: ProtocolPoolSelection;
+  v3PoolSelection?: ProtocolPoolSelection;
+  maxSwapsPerPath?: number;
+  maxSplits?: number;
+  distributionPercent?: number;
+}
+
+export const QUOTE_SPEED_MAP: { [key: string]: QuoteSpeedConfig } = {
```

Comment:
https://github.com/Uniswap/routing-api/blob/98c8fbeabb8c10c93e34e2c5f2ae0d4966595805/lib/handlers/shared.ts#L16

Probably makes sense to have all the routing config in one file?

Reply:
I usually prefer having smaller files that are easier to read, but since we already have the pattern of having all config in one file, I guess I can follow that here.